apps/visualneuro/packaging/packaging.cmake

```diff
@@ -31,6 +31,14 @@ set(CMAKE_INSTALL_OPENMP_LIBRARIES TRUE)
 set(CMAKE_INSTALL_SYSTEM_RUNTIME_COMPONENT Application)
 include (InstallRequiredSystemLibraries)
 
+if(APPLE)
+    # Fix for CPack Error on MacOS 13
+    # Error generating temporary disk image.
+    # hdiutil: create failed - Resource busy
+    # https://github.com/actions/runner-images/issues/7522#issuecomment-1564467252
+    set(CPACK_COMMAND_HDIUTIL "/usr/bin/sudo /usr/bin/hdiutil")
```

Comment:
Nice, we might need that one in inviwo too... I thought the runner just ran out of space I think...